definitions/control-v1.yaml

```diff
@@ -837,46 +837,46 @@ paths:
           content:
             application/json:
               schema:
-                "$ref": "#/components/schemas/error"
+                '$ref': '#/components/schemas/error'
         '401':
           description: Authentication failed
           content:
             application/json:
               schema:
-                "$ref": "#/components/schemas/error"
+                '$ref': '#/components/schemas/error'
         '404':
           description: App not found
           content:
             application/json:
               schema:
-                "$ref": "#/components/schemas/error"
+                '$ref': '#/components/schemas/error'
         '500':
           description: Internal server error
           content:
             application/json:
               schema:
-                "$ref": "#/components/schemas/error"
+                '$ref': '#/components/schemas/error'
         '503':
           description: 503 Service unavailable
           content:
             application/json:
               schema:
-                "$ref": "#/components/schemas/error"
-  "/apps/{app_id}/rules":
+                '$ref': '#/components/schemas/error'
+  '/apps/{app_id}/rules':
     get:
       summary: Lists Reactor rules
-      description: Lists the rules for the application specified by the application ID.
+      description: Lists the <a href="https://ably.com/documentation/general/queues#setup">rules</a> for the application specified by the application ID.
```

Comment:
This links to queue rules. I think we want to link to here: https://ably.com/documentation/general/events that is, not queue-specific rules.

Reply:
Noted. Removed the formatting too. (Must set a Prettier rule in my IDE!)